opentech/apply/funds/views.py

```diff
@@ -345,6 +347,35 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
+@method_decorator(staff_required, name='dispatch')
+class CreateProjectView(DelegatedViewMixin, CreateView):
+    context_name = 'project_form'
+    form_class = CreateProjectForm
+    model = Project
+
+    def form_valid(self, form):
+        submission_id = form.cleaned_data['submission']
+        try:
+            submission = ApplicationSubmission.objects.get(pk=submission_id)
+        except ApplicationSubmission.DoesNotExist:
```

Comment:
See comment on `projects/forms.py` I think we can rely on the form  to validate without throwing invaid ourselves

Reply:
Fixed in 379d6b5d